stubs/passlib/passlib/utils/handlers.pyi

```diff
@@ -183,3 +183,25 @@ class PrefixWrapper:
     def encrypt(self, secret, **kwds): ...
     def hash(self, secret, **kwds): ...
     def verify(self, secret, hash, **kwds): ...
+
+__all__ = [
+    # helpers for implementing MCF handlers
+    "parse_mc2",
+    "parse_mc3",
+    "render_mc2",
+    "render_mc3",
+    # framework for implementing handlers
+    "GenericHandler",
+    "StaticHandler",
+    "HasUserContext",
+    "HasRawChecksum",
+    "HasManyIdents",
+    "HasSalt",
+    "HasRawSalt",
+    "HasRounds",
+    "HasManyBackends",
+    # other helpers
+    "PrefixWrapper",
+    # TODO: a bunch of other things are commonly assumed in this namespace
+    #       (e.g. HEX_CHARS etc); need to audit uses and update this list.
```

Comment:
This is an upstream's TODO. I don't personally mind too much keeping the comments in `__all__` from a quick copy-paste, but that one really confused me.

```suggestion
```